File: Agilité/Verrocchio/Eleve.java

public class Eleve
{
    // variables d'instance - remplacez l'exemple qui suit par le vôtre
    private String nom;
    private int anneeNaissance;
    private Verrochio atelier;
    private Splinter master;
    /**
     * Constructeur d'objets de classe Leonardo
     */
    public Eleve(String nom, int anneeNaissance,Verrochio atelier, Splinter master) {
        this.nom = nom;
        this.anneeNaissance = anneeNaissance;
        this.atelier = atelier;
        this.master = master;
    }

    public void dessiner() {
        System.out.println(this.nom + " dessine dans l'atelier " + this.atelier.getNom());
    }
    public void apprentissage() {
        System.out.println(this.nom +" apprend à dessiner dans l'atelier "+ this.atelier.getNom());
    }

}

File: Eleve.py
import Music


class Eleve:
    Eleve = None

    def __init__(self, nom, anneeNaissance, atelier, master):
        self.nom = nom
        self.anneeNaissance = anneeNaissance
        self.atelier = atelier
        self.master = master
        self.music = Music
        #self.album= Album    

    def getNom(self):
        return self.nom

    def setNom(self, nom):
        self.nom = nom
        
    def dessiner(self):
        return self.nom + " dessine dans l'atelier " + self.atelier.getNom()

    def apprentissage(self):
        return self.nom + " apprend à dessiner dans l'atelier " + self.atelier.getNom()

    def prepare_couleur(self):
        return self.nom + " prepare a couleur pour "+ self.master.getNom()

 # ---------------------------------------------------------------------- Partie music
    def ecoute_music(self, music):
        return self.nom + "écoute la music "+ self.music.getNom(music)

    def ecoute_music_atelier(self):
        self.atelier.ecouter_music(self.music)


def apprentissage():
    return apprentissage()

def prepare_couleur(aritist):
    return prepare_couleur(aritist.nom)
